config/routes.rb

```diff
@@ -0,0 +1,21 @@
+Rails.application.routes.draw do
+  # get 'static/index'
+  # get 'pages/index'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] You should not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.